### DISCUSSION TITLE
Foundry 101 - Fund Me - issue with file directory results in error

### DISCUSSION THREAD

**Discussion by ARomano79** [Cyfrin/foundry-full-course-cu#1557] [General]
Hello, 
this is my first post here and I wanted to share a finding related to the Foundry 101 Fund Me project.

Within the first few videos for the project, Patrick specifies that we should grab our code from the Remix version of FundMe.sol to get started, and that we would be editing it later on.  My version threw an error related to the AggregatorV3Interface file once I started trying to create and test my scripts.  If this is happening, look for the `/shared` directory and remove it.  I found the error by doing a side by side check with a version shared by an admin.

![Screenshot shared error](https://github.com/Cyfrin/foundry-full-course-f23/assets/97197239/e21d04b2-2819-40c3-821e-4c77f102cc9d)

**Comment by EngrPips** [comment]
Hello @ARomano79 , Thanks for posting this on here to help people that might also run into this issue.

**Reply by ARomano79** [reply]
Anytime @EngrPips.  I hope it helps other students.  Also, I think this is my first non-user error 😆

**Reply by EngrPips** [reply]
Awesome, you are doing great !
